scripts/joinmarket-qt.py

```diff
@@ -1462,10 +1462,11 @@ def create_menu(self, position):
             txt = str(item.text(0))
             if validate_address(txt)[0]:
                 address_valid = True
-            if "EXTERNAL" in txt:
-                parsed = txt.split()
-                if len(parsed) > 1:
-                    xpub = parsed[1]
+
+            parsed = txt.split()
+            if len(parsed) > 1:
+                if parsed[-1].startswith("xpub") or parsed[-1].startswith("ypub") or parsed[-1].startswith("zpub"):
```

Comment:
Should we add `tpub` to this list? (which feeds into my comment on the next line, about repeated check here).